app/views/forms/edit/edit-capacityBuildingInitiative.js

```diff
@@ -22,54 +23,49 @@ import 'services/main';
-                  return countryRegions;
+
+              cbdSubjects    : function() { return $http.get("/api/v2013/thesaurus/domains/CBD-SUBJECTS/terms", { cache: true }).then(function(o){
+                var subjects = ['CBD-SUBJECT-BIOMES', 'CBD-SUBJECT-CROSS-CUTTING'];
+                var items = [];
+                _.forEach(subjects, function(subject) {
+                  var term = _.find(o.data, {'identifier': subject } );
+                  items.push(term);
+                  _(term.narrowerTerms).forEach(function (term) {
+                    items.push(_.find(o.data, {'identifier':term}));
+                  })
                 });
-              },           
-            // need to verify, where we are using this or any other future use ?
-            // chmregions: function() { return $q.all([thesaurusService.getDomainTerms('countries'), thesaurusService.getDomainTerms('geographicRegions')])
-            //     .then(function(data) {
-            //         return Enumerable.from($filter('orderBy')(data[0], 'title.en')).union(
-            //             Enumerable.from($filter('orderBy')(data[1], 'title.en'))).toArray();
-            //     });
-            // },
+                return items;
+              });
+            },
+            
         });
+    }, 100 );
+
     $scope.isGlobalOrRegional = function () {
-        if($scope.document && $scope.document.geographicScope && $scope.document.geographicScope.scope){
-            return _.includes(['56B8CEB7-56B5-436B-99D9-AA7C4622F326', 'C7D6719B-8AD9-4EB1-A472-B0B858DE0F56'], $scope.document.geographicScope.scope.identifier);
+        if($scope.document && $scope.document.geographicScope){
+            return _.includes(['56B8CEB7-56B5-436B-99D9-AA7C4622F326', 'C7D6719B-8AD9-4EB1-A472-B0B858DE0F56'], $scope.document.geographicScope.identifier);
         }
         return false;
     };
 
     $scope.isNational = function () {
-        if($scope.document && $scope.document.geographicScope && $scope.document.geographicScope.scope){
-            return $scope.document.geographicScope.scope.identifier == "20B2CC6D-646D-4FD5-BD53-D652BA3FA088";
+        if($scope.document && $scope.document.geographicScope){
+            return $scope.document.geographicScope.identifier == "20B2CC6D-646D-4FD5-BD53-D652BA3FA088";
         }
         return false;
     };
     //============================================================
     //
     //============================================================
     $scope.isSubnational = function () {
-        if($scope.document && $scope.document.geographicScope && $scope.document.geographicScope.scope){
-            return $scope.document.geographicScope.scope.identifier == "DEBB019D-8647-40EC-8AE5-10CA88572F6E";
+        if($scope.document && $scope.document.geographicScope){
+            return $scope.document.geographicScope.identifier == "DEBB019D-8647-40EC-8AE5-10CA88572F6E";
         }
         return false;
```

Comment:
we should slowly start using the new syntax for eg.
```suggestion
        return $scope.document?.geographicScope?.identifier == "DEBB019D-8647-40EC-8AE5-10CA88572F6E";
```